src/components/footer/footer.component.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import footerLogo from '../../assets/footer-logo.svg';
+import fbLogo from '../../assets/f_logo_white.svg';
+import twitterLogo from '../../assets/twitter_logo_white.svg';
+
+import './footer.styles.scss';
+
+const scrollToTop = () => {
+    window.scrollTo(0, 0);
+};
+
+const Footer = () => (
+    <div className="footer-background">
+        <div className="footer-top-section">
```

Comment:
should probably match this with the bottom section by adding or removing `-section`

Reply:
changed